third_party/move/tools/move-coverage/src/coverage_map.rs

```diff
@@ -68,7 +68,7 @@ impl CoverageMap {
         for line in BufReader::new(file).lines() {
             let line = line.unwrap();
             let mut splits = line.split(',');
-            let exec_id = splits.next().unwrap();
+            let exec_id = "exec_id";
```

Comment:
Use a fake key so that the data structure of the coverage map does not need to be changed